package.json

```diff
@@ -48,17 +48,16 @@
     }
   },
   "files": [
-    "core/dist/**",
-    "infinite/dist/**",
-    "immutable/dist/**",
-    "mutation/dist/**",
-    "_internal/dist/**",
+    "core/dist/**/*.{js,d.ts,mjs}",
```

Comment:
I think we can leverage gitignore?

https://docs.npmjs.com/cli/v8/using-npm/developers#keeping-files-out-of-your-package
> If there's no .npmignore file, but there is a .gitignore file, then npm will ignore the stuff matched by the .gitignore file. 

Maybe only leave the folder instead of appending `/**/*` to the folder.

```
"infinite/dist",
"immutable/dist"
```

Reply:
> "infinite/dist",
"immutable/dist"

I tried this pattern. Sadly it didn't work 😔

Reply:
How did you test it? It also doesn't work with `*/**/*.tsbuildinfo`?

Reply:
my config
```
{
   files: [
    "core/dist",
    "infinite/dist",
    "immutable/dist",
    "mutation/dist",
    "_internal/dist",
    "core/package.json",
    "infinite/package.json",
    "immutable/package.json",
    "mutation/package.json",
    "_internal/package.json",
  ]
}
```

```
dist
*/**/*.tsbuildinfo
```

Reply:
Tested a bit with `pnpm pack` and they're still included in the pack. Probably related to the `dist` is in both gitignore and package.json files field. Let's go with the current solution since we don't need to create extra .npmignore file